3_javascript/javascript1/week2/script5.js

```diff
@@ -0,0 +1,44 @@
+
+// Candy
+
+
+const candyPrices = {
+    sweet: 0.5,
+    chocolate: 0.7,
+    toffee: 1.1,
+    'chewing-gum': 0.03,
+  };
+  
+  const boughtCandyPrices = [];
+  let amountToSpend = Math.random() * 100;
+  
+  function addCandy(candyType, weight) {
+    const pricePerGram = candyPrices[candyType];
+    if (pricePerGram !== undefined) {
+      const totalPrice = pricePerGram * weight;
+      boughtCandyPrices.push(totalPrice);
+      amountToSpend -= totalPrice;
+      console.log('Added ${weight} grams of ${candyType} candy to your cart.');
```

Comment:
To allow using variables you need to use backquotes again :) Otherwise you will see this output `You need to move this condition up. Otherwise, because of the return statement in this function, it won't reach the queen.`